addusers.py

```diff
@@ -285,7 +256,13 @@ def parse_rows(rows):
     return projects, bad_rows
 
 if __name__ == "__main__":
-    openstack = Openstack(admin_user, admin_pwd, admin_tenant, auth_url, nova_version)
+    auth = v3.Password(auth_url=auth_url,
+                       username=admin_user,
+                       user_domain_id = 'default',
+                       password=admin_pwd)
```

Comment:
Not sure if you need a project_id here. Or project_name + project_domain_id (which is "default") Kristi would know this better than me. 

https://docs.openstack.org/developer/python-keystoneclient/using-sessions.html